.github/workflows/examples.yml

```
@@ -101,3 +101,9 @@ jobs:
         done
       env:
         OMP_NUM_THREADS: 1
+    - name: Run PyTorch snapshot example
```

[nit-picking] how about checkpoint instead of snapshot for the consistency?
```suggestion
    - name: Run PyTorch checkpoint example
```

---

Thanks! I fixed.